src/layout/DetailedTimer.tsx

```diff
@@ -25,6 +30,10 @@ export default class DetailedTimer extends React.Component<Props> {
         }
 
         const totalHeight = this.props.state.timer.height + this.props.state.segment_timer.height;
+        const topRowHeight = totalHeight * 0.55;
+        const bottomRowHeight = totalHeight - topRowHeight;
+        const topRowFontSize = topRowHeight * 0.5 * fontSizeToLineHeightRatio;
```

Comment:
Also why don't we need to parse them here? Are we multiplying strings here?!

Reply:
I think it's implicitly converting the string to a number. We should definitely just do the conversion when we import the variable though.